taskfile/read/taskfile.go

```diff
@@ -68,24 +74,27 @@ func Taskfile(dir string, entrypoint string) (*taskfile.Taskfile, error) {
 			return err
 		}
 		if !filepath.IsAbs(path) {
-			path = filepath.Join(dir, path)
+			path = filepath.Join(readerNode.Dir, path)
 		}
 
-		path, err = exists(path)
-		if err != nil {
-			if includedTask.Optional {
-				return nil
-			}
+		includeReaderNode := &ReaderNode{
+			Dir:        filepath.Dir(path),
+			Entrypoint: filepath.Base(path),
```

Comment:
Seems that we have the wrong value here when inclusion is by a directory (omits `Taskfile.yml`).

Reply:
Fixed! I added a check for `exists()` so all default taskfile names can be used 😄